### PR TITLE
Add an issue template

### PR DESCRIPTION
**Pull request by adam-fowler**
Adding issue template so we don't need to keep asking the same questions when an issue comes through

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (Yasumoto) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
about: Create a report to help us improve the functionality of AWSSDKSwift!
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (Yasumoto) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
A clear and concise description of what the bug is. If you can include a link to redacted requests or responses, that would be very helpful as well!
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Setup (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Version of aws-sdk-swift [e.g. 3.0]
```
Comment (Yasumoto) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
 - Version of aws-sdk-swift [e.g. 3.0]
 - Authentication mechanism [hard-coded credentials, IAM Instance Profile on EC2, etc]
```

**Comment by adam-fowler** [comment]
@Yasumoto I'll add the same for aws-sdk-swift-core

**Pull request merged**